SU-HSDO/suhumsci
# Arch 81 postcard paragraph

# READY FOR REVIEW

# Summary
- Postcard paragraph type with 2 view modes.

# Needed By (Date)
- ASAP

# Urgency
- Low

# Steps to Test

1. Checkout this branch
2. sync to production
3. enable hs_postcard module
4. create basic page with postcard paragraph
5. ensure the display field switches it from horizontal to vertical.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
